spec/factories/ext_management_system.rb

```diff
@@ -3,12 +3,12 @@
     zone { EvmSpecHelper.local_miq_server.zone }
 
     after(:build) do |ems|
-      ems.hostname = Rails.application.secrets.vmware_cloud[:host]
+      ems.hostname = vcr_secret_by_key_path(:vmware_cloud, :host)
     end
 
     after(:create) do |ems|
-      userid   = Rails.application.secrets.vmware_cloud[:userid]
-      password = Rails.application.secrets.vmware_cloud[:password]
+      userid   = vcr_secret_by_key_path(:vmware_cloud, :userid)
+      password = vcr_secret_by_key_path(:vmware_cloud, :password)
```

Comment:
If we change the method names as per my comment in the core PR, then this becomes, which I think is even more readable (or at least less wordy)

```suggestion
      userid   = vcr_secret(:vmware_cloud, :userid)
      password = vcr_secret(:vmware_cloud, :password)
```